### PR TITLE
fix: ListingBooking

### PR DESCRIPTION
**Pull request by Mdjuealhossain**

### CODE REVIEW

File: src/widgets/ListingDetailsBanner/index.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { FC } from "react";
 import { Paper, Typography } from "@mui/material";
 
-import { ListingDetailsBannerProps } from "./Types";
+import { ListingBookingProps } from "./Types";
 
-const ListingDetailsBanner: FC<ListingDetailsBannerProps> = () => {
+const ListingDetailsBanner: FC<ListingBookingProps> = () => {
```
Comment (arifulbgt4) [inline src/widgets/ListingDetailsBanner/index.tsx:6]: Component name PropsType name should same

**Review by arifulbgt4 — CHANGES_REQUESTED** [review]

File: src/widgets/ListingBooking/index.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  Divider,
+  Button,
+  ButtonBase,
+} from "@mui/material";
+import ShoppingCartIcon from "@mui/icons-material/ShoppingCart";
+import PolylineRoundedIcon from "@mui/icons-material/PolylineRounded";
+
+import { ListingDetailsSidebarProps } from "./Types";
+
+const ListingBooking: FC<ListingDetailsSidebarProps> = () => {
```
Comment (arifulbgt4) [inline src/widgets/ListingBooking/index.tsx:17]: Component name And Props type name should same

**Review by arifulbgt4 — CHANGES_REQUESTED** [review]

File: src/widgets/ListingDetailsBanner/index.tsx (hunk 1 above)
Comment (Mdjuealhossain) [reply src/widgets/ListingDetailsBanner/index.tsx:6]: solved

File: src/app/[locale]/l/page.tsx [hunk 3]
```diff
@@ -1,24 +1,28 @@
-import { Grid } from "@mui/material";
+import { Box, Container, Grid } from "@mui/material";
 
 import Listings from "src/widgets/Listings";
 import ListingsSidebarFilter from "src/widgets/ListingsSidebarFilter";
 import SearchFilter from "src/widgets/SearchFilter";
 
 const ListingAll = () => {
```
Comment (arifulbgt4) [inline src/app/[locale]/l/page.tsx:7]: Component: ListingPage

**Review by arifulbgt4 — CHANGES_REQUESTED** [review]

**Review by arifulbgt4 — APPROVED** [review]

**Pull request merged**